docs/template_plugin/CMakeLists.txt

```diff
@@ -11,6 +11,8 @@ set(TEMPLATE_PLUGIN_SOURCE_DIR ${OpenVINOTemplatePlugin_SOURCE_DIR})
 
 find_package(OpenVINODeveloperPackage REQUIRED)
 
+ov_option(ENABLE_TEMPLATE_REGISTRATION "Enables registration of TEMPLATE plugin" OFF)
```

Comment:
why not in dedicated file https://github.com/openvinotoolkit/openvino/blob/master/cmake/features.cmake#L107 ?